overlord/snapstate/storehelpers.go

```diff
@@ -657,8 +669,14 @@ func refreshCandidates(ctx context.Context, st *state.State, names []string, use
 		}
 
 		if !snapst.IgnoreValidation {
-			if enforcedSets != nil {
-				requiredValsets, requiredRevision, err := enforcedSets.CheckPresenceRequired(naming.Snap(installed.InstanceName))
+			var requiredValsets []snapasserts.ValidationSetKey
```

Comment:
should name be `requiredValSets`?

Reply:
I don't see this acted on, in line 808 you use `requiredValSets`

Reply:
The previous code already used both and `valset` is the most common but I can make add a commit to rename them